components/browser/menu/src/main/java/mozilla/components/browser/menu/item/WebExtensionBrowserMenuItem.kt

```diff
@@ -83,4 +91,38 @@ class WebExtensionBrowserMenuItem(
         labelView.invalidate()
         badgeView.invalidate()
     }
+
+    @Suppress("TooGenericExceptionCaught")
```

Comment:
Nit: why is `Throwable` an appropriate exception here?  It doesn't look like you follow this policy in AC, but on FFTV any suppression needed an explanatory comment.  If it didn't need one, then the rule should probably be globally relaxed.

Reply:
In this case I just copied the code that was already used. @psymoon Can you comment here?

Reply:
@Baron-Severin @NotWoods generic throwable is fine here since this is for a fallback option like an else branch.